pkg/routeagent_driver/handlers/ovn/uninstall.go

```diff
@@ -60,10 +60,8 @@ func (ovn *Handler) Stop(uninstall bool) error {
 			constants.RouteAgentHostNetworkTableID, err)
 	}
 
-	err = ovn.cleanupForwardingIptables()
```

Comment:
So we don't need to run this?

Reply:
No. It only does partial cleanup which is necessary when there is a Gateway failover.